package.json

```diff
@@ -44,32 +44,32 @@
     "test:integration": "mocha test-js --require babel/register"
   },
   "dependencies": {
-    "browserify": "^16.2.3",
+    "browserify": "^16.5.1",
     "browserify-incremental": "^3.1.1",
     "concat-stream": "^2.0.0",
     "gaze": "^1.1.3",
-    "glob": "^7.1.3",
+    "glob": "^7.1.6",
     "mold-source-map": "^0.4.0",
     "node-static": "^0.7.11",
     "read": "^1.0.7",
     "sorcery": "^0.10.0",
-    "temp": "^0.9.0",
+    "temp": "^0.9.1",
     "through": "^2.3.8",
-    "tree-kill": "^1.2.1",
-    "which": "^1.3.1",
+    "tree-kill": "^1.2.2",
+    "which": "^2.0.2",
     "wordwrap": "1.0.0"
   },
   "devDependencies": {
-    "babel": "^5.8.22",
-    "bower": "^1.8.4",
+    "babel": "^6.23.0",
```

Comment:
These 'vulnerabilities' are completely irrelevant for a CLI app like Pulp, and I vaguely remember trying and failing to update to babel 6 at some point in the past because of some breaking change. I don't want to risk introducing new bugs so would you mind reverting these changes (the ones in package.json) please?

Reply:
Sure. This wouldn't have helped with the one it warns about when you install `pulp` anyway, unfortunately. It's a dependency of `node-serve` that it warns about just now, and that hasn't been touched for two years so I doubt it's going to get fixed any time soon.

Reply:
Ah ok, good to know, thanks!